tools/alerts/create_alerts.py

```diff
@@ -280,6 +285,63 @@ def get_recurrently_failing_jobs_alerts(
     return alerts
 
 
+def generate_viable_strict_lag_alert() -> List[Dict[str, Any]]:
+    # get viable strict lag from rockset
+    def get_viable_strict_lag() -> Any:
+        # Define the name of the Rockset collection and lambda function
+        collection = "metrics"
+        lambda_function_name = "strict_lag_sec"
+        lambda_function_version = "e0ab723990d6f2a2"
+        api_response = run_rockset_query(
+            collection, lambda_function_name, lambda_function_version
+        )
+        return api_response["results"][0]["strict_lag_sec"]
+
+    def during_weekend() -> bool:
+        # Get the pst datetime
+        PST_OFFSET = -8
+        now = datetime.now(timezone(offset=timedelta(hours=PST_OFFSET)))
+
+        # Define the time interval
+        start_time_fri = time(17, 0, 0)  # 5:00:00 PM
+        end_time_mon = time(6, 0, 0)  # 6:00:00 AM
+
+        # Check if current time is in the interval
+        if now.weekday() == 4:  # Friday
+            if now.time() >= start_time_fri:
+                return True
+        elif now.weekday() == 0:  # Monday
+            if now.time() < end_time_mon:
+                return True
+        elif now.weekday() in [5, 6]:  # Saturday and Sunday
+            return True
+        return False
+
+    if during_weekend():
+        return []
+
+    lag = int(get_viable_strict_lag())
+    if lag < MINIMUM_STRICT_LAG:
```

Comment:
The lag could be well below the threshold to be 0 when the viable/strict commit and main commit to be the same.  So I think we could remove the lower bound condition